### PR TITLE
Project import generated by Copybara.

### PR DESCRIPTION
**Pull request by trav3711**
Project import generated by Copybara.

### CODE REVIEW

File: tools/validators/instance_validator/README.md [hunk 1]
```diff
@@ -8,42 +8,10 @@ Installing and using the Instance Validator requires Python 3.9, and the specifi
 
 * MacOS & Linux: `python3`
 * Windows (with only Python 3 installed): `python` or `py -3`
-* Windows (with Python 3 installed): `py -3`
+* Windows (with Python 2 and 3 installed): `py -3`
 
 You can run the command with just the version flag (e.g. `python --version`) to verify that the result is `Python 3.*`.
 
-
```
Comment (charbull) [inline tools/validators/instance_validator/README.md:15]: this is deleting information we added to github

File: tools/validators/instance_validator/instance_validator.py [hunk 2]
```diff
@@ -96,14 +94,6 @@ def _ParseArgs() -> argparse.ArgumentParser:
       help='Filename for the validation report',
       metavar='report-filename')
 
-  parser.add_argument(
-      '--udmi',
```
Comment (charbull) [inline tools/validators/instance_validator/instance_validator.py:100]: those are being deleted

File: tools/validators/instance_validator/setup.py [hunk 3]
```diff
@@ -28,8 +28,10 @@
     'Nigel Kilmer',
     packages=find_packages(),
     install_requires=[
+        'protobuf==3.20.0rc2', 'proto-plus<1.19.7',
         'ruamel.yaml==0.17.4', 'strictyaml==1.4.2',
-        'google-cloud-pubsub', 'googleapis-common-protos',
-        'google-auth', 'protobuf', 'proto-plus'],
```
Comment (charbull) [inline tools/validators/instance_validator/setup.py:33]: keep those, I think thats why your tests are failing

File: tools/validators/instance_validator/tests/entity_instance_test.py [hunk 4]
```diff
@@ -1042,11 +1039,22 @@ def testValidate_GoodUpdateOperationDefaultExport_Success(self):
     self.assertEqual(entity_instances['PHYSICAL-ENTITY-GUID'].operation,
                      _EXPORT)
 
-  def testValidate_UdmiEntityPresentValue_Fails(self):
-    # the example used is a valid entity according to dbo; however, not udmi
```
Comment (charbull) [inline tools/validators/instance_validator/tests/entity_instance_test.py:1046]: keep those udmi ones

File: tools/validators/instance_validator/validate/entity_instance.py [hunk 5]
```diff
@@ -32,11 +31,6 @@
 _CONFIG_UPDATE = parse.ConfigMode.UPDATE
 _CONFIG_INIT = parse.ConfigMode.INITIALIZE
 _CONFIG_EXPORT = parse.ConfigMode.EXPORT
-# udmi present value: points.$name.present_value
```
Comment (charbull) [inline tools/validators/instance_validator/validate/entity_instance.py:35]: keep those udmi

File: tools/validators/instance_validator/validate/telemetry_validator.py [hunk 6]
```diff
@@ -34,11 +33,10 @@ class TelemetryValidator(object):
   Attributes;
     entities: a dict with entity_name as a key and EntityInstance as value.
     timeout: the max time the validator must read messages from pubsub.
-    is_udmi: true/false treat telemetry stream as UDMI
     callback: the method called by the pubsub listener upon receiving a msg.
   """
 
-  def __init__(self, entities, timeout, is_udmi, callback):
```
Comment (charbull) [inline tools/validators/instance_validator/validate/telemetry_validator.py:41]: this should stay and its tests.

**Review by charbull — CHANGES_REQUESTED** [review]
I think we need to bring the udmi and readme changes before we sync.
It seems we are out of sync here?

File: tools/validators/instance_validator/README.md (hunk 1 above)
Comment (trav3711) [reply tools/validators/instance_validator/README.md]: yup, reverted.

File: tools/validators/instance_validator/instance_validator.py (hunk 2 above)
Comment (trav3711) [reply tools/validators/instance_validator/instance_validator.py]: yup, reverted.

File: tools/validators/instance_validator/setup.py (hunk 3 above)
Comment (trav3711) [reply tools/validators/instance_validator/setup.py]: yup, reverted.  Also changed python version to >= 3.9

File: tools/validators/instance_validator/validate/entity_instance.py (hunk 5 above)
Comment (trav3711) [reply tools/validators/instance_validator/validate/entity_instance.py]: yup, reverted.

File: tools/validators/instance_validator/tests/entity_instance_test.py (hunk 4 above)
Comment (trav3711) [reply tools/validators/instance_validator/tests/entity_instance_test.py]: yup, reverted.

File: tools/validators/instance_validator/validate/telemetry_validator.py (hunk 6 above)
Comment (trav3711) [reply tools/validators/instance_validator/validate/telemetry_validator.py]: yup, reverted.

File: tools/validators/instance_validator/validate/subscriber.py [hunk 7]
```diff
@@ -57,7 +65,7 @@ def Listen(self, callback):
     else:
       print("No service account. Using application default credentials")
       # pylint: disable=unused-variable
-      credentials, project_id = auth.default()
```
Comment (charbull) [inline tools/validators/instance_validator/validate/subscriber.py:60]: this is deleting the correct one: https://google-auth.readthedocs.io/en/master/reference/google.auth.html

**Review by charbull — CHANGES_REQUESTED** [review]

File: tools/validators/instance_validator/validate/subscriber.py (hunk 7 above)
Comment (trav3711) [reply tools/validators/instance_validator/validate/subscriber.py]: Thanks for catching that. Fixed.

**Review by charbull — APPROVED** [review]

**Pull request merged**